src/workers/nlmExtractTables.ts

```diff
@@ -104,27 +104,26 @@ const nlmExtractTables = new DiscordWorker(
         return acc
       }, {} as Record<number, typeof results>)
 
-      const tables = await Promise.all(
-        Object.entries(groupedResults).map(async ([page_idx, tablesOnPage]) => {
-          const context = tablesOnPage.map(
-            (table) =>
-              `\n#### ${
-                table.name
-              } (Extracted table with more precision from PAGE ${page_idx})\n\n${
-                table.markdown || ''
-              }`
-          )
+      const tables = await Object.entries(groupedResults).reduce(
+        async (resultsPromise, [page_idx, tablesOnPage]) => {
+          const results = await resultsPromise
+          const lastPageMarkdown = results.at(-1)?.markdown || ''
           const filename = tablesOnPage[0].filename
           const markdown = await extractTextViaVisionAPI(
             { filename, name: `Tables from page ${page_idx}` },
-            context.slice(-3).join('\n')
+            lastPageMarkdown.slice(0, 5000)
```

Comment:
Why limit to 5000 characters here?

Does this mean that we send a larger context compared to before when we only sent the last 3 tables?